### DISCUSSION TITLE
Issue with separating classical music releases

### DISCUSSION THREAD

**Discussion by iagomago** [quodlibet/quodlibet#4469] [General]
The issue I'm having concerns the separating of different releases, especially the classical music ones, where the same music is being performed by the same musicians in a different time period (some examples for reference: Glenn Gould's version of The Goldberg Variations from [1956](https://www.discogs.com/master/261784-Bach-Glenn-Gould-The-Goldberg-Variations) and [1982](https://www.discogs.com/master/48325-Bach-Glenn-Gould-The-Goldberg-Variations), which get merged into a single release no matter how many "sort" tags I input). As of now, I haven't found a way to separate said releases whenever this issue arose. Maybe it could be fixed by adding the possibility to order your column library according to arbitrary tags instead of a list of predetermined ones?

**Comment by Npa972** [comment]
Hi why don't you rename the albums.
![Capture d’écran_2023-11-19_10-29-38](https://github.com/quodlibet/quodlibet/assets/49346431/458ffbab-97dd-4cef-9414-c77299def280)
This is in Rhythmbox but it works in QL.

**Comment by nemethf** [comment]
File > Preferences > Song List > Visible Columns > Edit 
then change "Album / Disc subtitle" from `album~discsubtitle` to `<~album~discsubtitle> <~year>`.  This will separate the albums in most browser-types.

Alternatively you can try to sort by musicbrainzartistid if you tag your music library with Picard, for example.   See: [1955](https://musicbrainz.org/release-group/dcff25f1-702d-3b5e-b0da-d48172e6e62a), [1981](https://musicbrainz.org/release-group/dcff25f1-702d-3b5e-b0da-d48172e6e62a).

**Comment by declension** [comment]
I guess it depends what you mean by "separate". Do you mean visually, or separate albums in the various album browsers?

Either way make sure you have a unique ID for the release / album, e.g. `labelid` or better `musicbrainz_albumid`. See [the FAQ entry on separating albums](https://quodlibet.readthedocs.io/en/latest/guide/faq.html#i-have-two-albums-with-the-same-name-which-are-merged-in-the-album-list).
